src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public interface Storage {
+    Map<String, Integer> storage = new HashMap<>();
+    void update(FruitTransaction fruitTransaction, int count);
+
+    int calculateAmount(FruitTransaction fruitTransaction);
+
+    List<String> getAll();
+}

```

Comment:
According to the principles of SOLID (namely Single Responsibility Principle) -  class or interface must be responsible for one function. I think, such decision is not ok. It's better to implement the map in a class and not an interface, and transfer the methods to separate interfaces.

Reply:
In principle, I agree with you, but I think that having too many different interfaces for such a task might lead to difficulties in test coverage. Let's wait for the mentor's opinion.